openshift-kube-apiserver/filters/deprecatedapirequest/apiaccess_count_controller_test.go

```diff
@@ -670,9 +685,14 @@ func TestPersistRequestCountForAllResources(t *testing.T) {
 			},
 		},
 	}
+	buf := &bytes.Buffer{}
+	klog.SetOutput(io.Discard)
+	klog.SetOutputBySeverity("ERROR", buf)
+	klog.LogToStderr(false)
```

Comment:
won't this affect other tests run as a part of this run ?
should we unset it?

also, I think I would remove it, checking a log output is not important. More important is to check if we panic and the content of `logContains`. WDYT?

Reply:
Update test. Removed log check and simply test now that we don't panic and other resources are updated correctly.